Npm build fails

Building from the latest main fails:

```
$ npm run build

> tribler-webui@0.1.0 build
> tsc && vite build

src/Router.tsx:63:27 - error TS2786: 'GeneralSettings' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.
    Type 'undefined' is not assignable to type 'Element | null'.

63                 element: <GeneralSettings />,
                             ~~~~~~~~~~~~~~~

src/Router.tsx:67:27 - error TS2786: 'Connection' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.

67                 element: <Connection />,
                             ~~~~~~~~~~

src/Router.tsx:71:27 - error TS2786: 'Bandwidth' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.

71                 element: <Bandwidth />,
                             ~~~~~~~~~

src/Router.tsx:75:27 - error TS2786: 'Seeding' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.

75                 element: <Seeding />,
                             ~~~~~~~

src/Router.tsx:79:27 - error TS2786: 'Anonymity' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.

79                 element: <Anonymity />,
                             ~~~~~~~~~

src/Router.tsx:83:27 - error TS2786: 'Debugging' cannot be used as a JSX component.
  Its return type 'Element | undefined' is not a valid JSX element.

83                 element: <Debugging />,
                             ~~~~~~~~~


Found 6 errors in the same file, starting at: src/Router.tsx:63
```

--- a/.github/workflows/pr-npm.yml
+++ b/.github/workflows/pr-npm.yml
@@ -1,0 +1,14 @@
+name: Build GUI
+on: pull_request
+jobs:
+  linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup npm
+        uses: actions/setup-node@v4
+      - name: Build npm
+        run: |
+          cd src/tribler/ui/
+          npm install
+          npm run build

--- a/src/tribler/ui/src/pages/Settings/Anonymity.tsx
+++ b/src/tribler/ui/src/pages/Settings/Anonymity.tsx
@@ -12,7 +12,7 @@ export default function Anonimity() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

--- a/src/tribler/ui/src/pages/Settings/Bandwidth.tsx
+++ b/src/tribler/ui/src/pages/Settings/Bandwidth.tsx
@@ -13,7 +13,7 @@ export default function Bandwith() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

--- a/src/tribler/ui/src/pages/Settings/Connection.tsx
+++ b/src/tribler/ui/src/pages/Settings/Connection.tsx
@@ -15,7 +15,7 @@ export default function Connection() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

--- a/src/tribler/ui/src/pages/Settings/Debugging.tsx
+++ b/src/tribler/ui/src/pages/Settings/Debugging.tsx
@@ -12,7 +12,7 @@ export default function Debugging() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

--- a/src/tribler/ui/src/pages/Settings/General.tsx
+++ b/src/tribler/ui/src/pages/Settings/General.tsx
@@ -15,7 +15,7 @@ export default function General() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

--- a/src/tribler/ui/src/pages/Settings/Seeding.tsx
+++ b/src/tribler/ui/src/pages/Settings/Seeding.tsx
@@ -14,7 +14,7 @@ export default function Seeding() {
 
     if (!settings) {
         (async () => { setSettings(await triblerService.getSettings()) })();
-        return;
+        return null;
     }
 
     return (

